cmd/minikube/cmd/start.go

```diff
@@ -182,6 +198,19 @@ func runStart(cmd *cobra.Command, args []string) {
 		cmdutil.MaybeReportErrorAndExit(err)
 	}
 
+	var rtcfg = map[string]string{}
+	containerRuntime := viper.GetString(containerRuntime)
+	if rtcfg := SetContainerRuntime(rtcfg, containerRuntime); rtcfg != nil {
+		var command string
+		fmt.Println("Writing crictl config...")
```

Comment:
Is this an accurate statement if containerd is being used?

Reply:
Sure, CRI (the standard) is different from CRI-O (the implementation)

https://kubernetes.io/docs/setup/cri/

We don't bother with it for docker, but technically it is used even there...

`/var/run/dockershim.sock`